docs/apache-airflow-providers-amazon/operators/emr/emr.rst

```diff
@@ -189,6 +189,11 @@ Wait on an Amazon EMR job flow state
 To monitor the state of an EMR job flow you can use
 :class:`~airflow.providers.amazon.aws.sensors.emr.EmrJobFlowSensor`.
 
+This sensor can be run in deferrable mode by passing ``deferrable=True`` as a parameter.
+Using ``deferrable`` mode will release worker slots and leads to efficient utilization of
+resources within Airflow cluster.However this mode will need the Airflow triggerer to be
+available in your deployment.
+
```

Comment:
I don't think it make sense to have this generic information about the benefits of deferrable operators everywhere. This document should just let the user know that deferrable mode is available. We have other documentation that highlight the benefits of deferrable operators.

Reply:
I kind of agree but added it after the comment https://github.com/apache/airflow/pull/32029#pullrequestreview-1499885125. removed it